app/lib/coronavirus_local_restrictions_constraint.rb

```diff
@@ -0,0 +1,7 @@
+class CoronavirusLocalRestrictionsConstraint
+  def matches?(*)
+    return false if Rails.env.production?
```

Comment:
Nit, could be `!Rails.env.production?` but don't worry about changing given how short lived this might be.